barretenberg/cpp/src/barretenberg/vm/avm_trace/AvmMini_opcode.hpp

```diff
@@ -4,7 +4,6 @@
 #include <unordered_map>
 
 namespace avm_trace {
-using std::size_t;
```

Comment:
If this was failing in CI maybe you were missing <cstddef> or some other header that actually defines size_t.  You may be getting the definition of std::size_t indirectly through <unordered_map>.

Reply:
Yep. I will add <cstddef>